roofit/roofit/src/RooBukinPdf.cxx

```diff
@@ -16,24 +16,26 @@
 /** \class RooBukinPdf
     \ingroup Roofit
 
-The RooBukinPdf implements the NovosibirskA function. For the parameters, see
+RooBukinPdf implements the NovosibirskA function. For the parameters, see
 RooBukinPdf().
 
 Credits:
 May 26, 2003.
-A. Bukin, Budker INP, Novosibirsk
+A.Bukin, Budker INP, Novosibirsk
 
-\image html RooBukin.png
+http://www.slac.stanford.edu/BFROOT/www/Organization/CollabMtgs/2003/detJuly2003/Tues3a/bukin.ps
 **/
 
 #include "RooFit.h"
 
-#include <math.h>
-
+#include <cmath>
 
```

Comment:
Why this? Does it change anything?

Reply:
Yes, .h files are legacy. They declare everything in global scope. They are for compatibility with C, because C doesn't have namespaces and overloads.

https://stackoverflow.com/questions/8734230/math-interface-vs-cmath-in-c